feature/session/src/main/res/layout/fragment_bottom_sheet_favorite_session.xml

```diff
@@ -57,7 +57,7 @@
             android:text="@string/start_filter"
             android:textAppearance="@style/TextAppearance.DroidKaigi.Body2"
             app:icon="@drawable/ic_filter_list_black_24dp"
-            app:isHide="@{(!isEmptySessions || isFiltered) &amp;&amp; !isCollapsed}"
+            app:isInvisible="@{(isEmptySessions || !isFiltered) &amp;&amp; isCollapsed}"
```

Comment:
Could you please change `inverted value` to `suggestion value` ?

```suggestion
            app:isInvisible="@{(isEmptySessions &amp;&amp; !isFiltered) || isCollapsed}"
```

Reply:
You are right! I fix it!

Reply:
I fixed it🙆‍♂️